apis/v1alpha1/bpfApplication_types.go

```diff
@@ -42,15 +42,9 @@ const (
 	// ProgTypeKprobe refers to the Kprobe program type.
 	ProgTypeKprobe EBPFProgType = "Kprobe"
 
-	// ProgTypeKretprobe refers to the Kprobe program type.
-	ProgTypeKretprobe EBPFProgType = "Kretprobe"
-
 	// ProgTypeUprobe refers to the Uprobe program type.
```

Comment:
I don't know why u deleted the kproberet and uproberet program types ?

Reply:
They weren't being used.